src/connect.js

```diff
@@ -20,7 +20,7 @@
  * @return {Function}
  */
 function connect(params) {
-  // write code here
+  return (extraParams) => ({ ...params, ...extraParams });
```

Comment:
no need for parentheses around inner function argument (it's only one)

Reply:
> no need for parentheses around inner function argument (it's only one)

i  return not a function - i return an object. This parentheses are necessary in this case. Whithout parentheses will be error because without parentheses function body expected.

Reply:
> > no need for parentheses around inner function argument (it's only one)
> 
> i return not a function - i return an object. This parentheses are necessary in this case (when arrow function return object - parentheses must be). Whithout parentheses will be error because without parentheses function body expected.

Reply:
I didn't say anything about function body, it's about argument

Reply:
> I didn't say anything about function body, it's about argument

fixed